src/main/kotlin/br/com/creditas/projethom/model/System.kt

```diff
@@ -0,0 +1,20 @@
+package br.com.creditas.projethom.model
+
+import java.time.LocalDateTime
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.ManyToOne
+
+@Entity
+data class System(
+  @Id
+  val id: UUID = UUID.randomUUID(),
+  val name: String,
+  @ManyToOne
+  val owner: Team? = null,
+  val documentation: String,
+  val url: String,
+  val createdAt: LocalDateTime = LocalDateTime.now(),
+  var updatedAt: LocalDateTime = LocalDateTime.now()
```

Comment:
its works, but is usual to use a column annotation to describe the name. But its ok